src/js/components/FileInput/doc.js

```diff
@@ -51,6 +51,9 @@ export const doc = FileInput => {
         number of files will be shown.`,
       )
       .defaultValue(undefined),
+    maximumCount: PropTypes.number
```

Comment:
It doesn't seem like this is being used. We should remove and update the multiple docs to show that max is an option

Reply:
Done